### PR TITLE
Footer area

### PR DESCRIPTION
**Pull request by kilonzi**
Request for review from the last branch; 

- not tested for different screen sizes
- no media queries done on the project

### CODE REVIEW

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

File: index.html
Comment (Torres-ssf) [inline index.html]: [OPTIONAL]
Use the `nav` tag for the navbar instead of `div` tag. Div tags have no semantic meaning. Semantic HTML is a very important tag. You can read more [here](http://seekbrevity.com/semantic-markup-important-web-design/).

File: index.html
Comment (Torres-ssf) [inline index.html]: [OPTIONAL]
You should use the `main` tag for the most relevant content on your page. This is related to semantic HTML.

File: index.html
Comment (Torres-ssf) [inline index.html:49]: [OPTIONAL]
You can  use a `section` tag instead here

File: index.html
Comment (Torres-ssf) [inline index.html]: **[REQUIRED]**
It's best practice to use the `footer` tag for the footer. This also related to semantic HTML.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Hi there @kilonzi :hand: 
Nice work build this page :smile: 
But before I give you :+1:, there are some changes that we need to make:

[REQUIRED]
- [x]  There are some errors in your HTML code. Check them using this HTML [validator](https://validator.w3.org/) . Ignore the warnings.

[REQUIRED]
- [x] There is a problem in the way you are loading images, some of then are not showing. 
Screenshot from 2019-08-30 08-52-15

[REQUIRED]
- [x] The footer is on top of the promo section
Screenshot from 2019-08-30 08-55-05

[FRIENDLY-REMAINDER]
For future projects, use only one feature branch for the HTML/CSS projects and do not merge any feature branch to master until it's been approved.

Please kindly make these changes and submit for code review again.
Happy coding :computer:

File: index.html
Comment (kilonzi) [reply index.html]: I have changed to `nav` to be semantic.

**Comment by kilonzi** [comment]
> ### Hi there @kilonzi ✋
> Nice work build this page 😄
> But before I give you 👍, there are some changes that we need to make:
> 
> [REQUIRED]
> 
> * [x]   There are some errors in your HTML code. Check them using this HTML [validator](https://validator.w3.org/) . Ignore the warnings.
> 
> [REQUIRED]
> 
> * [x]  There is a problem in the way you are loading images, some of then are not showing.
>   Screenshot from 2019-08-30 08-52-15
> 
> [REQUIRED]
> 
> * [x]  The footer is on top of the promo section
>   Screenshot from 2019-08-30 08-55-05
> 
> [FRIENDLY-REMAINDER]
> For future projects, use only one feature branch for the HTML/CSS projects and do not merge any feature branch to master until it's been approved.
> 
> Please kindly make these changes and submit for code review again.
> Happy coding 💻

Resolved all the issues; I have an issue with screen sizes (responsiveness)  but I will work on it

File: style.css
Comment (samgaco) [inline style.css:134]: Using different units could help with adapting the website to different screen sizes.

**Review by samgaco — CHANGES_REQUESTED** [review]
#### Hi @kilonzi :hand: 

#### Everything is looking good, some final things before proceeding:

* Try to make the footer look closer to the one in the original website:

> footer


* the four "new-products" boxes should be linked

* Maybe the fact that you used "px" for some measures is giving problems in different screens, for example in mine you can see this. Maybe [max-width ](https://developer.mozilla.org/es/docs/Web/CSS/max-width) could work better for this cases.

> apple

### Please :pray: work on these points and submit back for another code review when you are done. Regards :smile:

File: style.css
Comment (kilonzi) [reply style.css:134]: I have worked on:-
- Updating some sections with **max-width** has resolved most of the issues (except mobile). Thanks for pointing me to the link
- Added links to the products
- Changed the footer margins and line spacing to look like than the original site

`Let me know if there is anything else to work on`

**Comment by Oghenebrume50** [comment]
Reviewing.......

File: index.html
Comment (Oghenebrume50) [inline index.html]: [OPTIONAL] You may not need form here since there is only one input element

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Hi @kilonzi  good job mate, you have made good progress so far, here are some outlined this to correct, currently the search input display is off, and there is still a margin at the top, please fix that. you may want to look into CSS specificity before using !important. Please fix that and request a re-review.

File: index.html
Comment (kilonzi) [reply index.html]: I have removed the form element 
[Resolved] the "search input field off"

File: style.css
Comment (bolah2009) [inline style.css]: #### `REQUIRED`

### Let's remove this:

```css
    margin-top: 0 !important;
    padding-top: 0 !important;
```
1. It's rudundant, since we have `margin: 0;` and `padding: 0;` in `css-reset.css` file. (See image in next comment)
2. Always look for a way to use [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) before even considering `!important`. 

> Make sure you DON’T make [common mistakes](https://speckyboy.com/good-bad-css-practices/) in your CSS styles.

> **Recommendation:** You should only use the `!important` rule when overriding styles in a user style sheet, overriding 3rd party code and inline styles as well as utility classes.


### Learn more about [`!important` here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception)

File: style.css
Comment (bolah2009) [inline style.css]: #### `REQUIRED`

Check the comment on using redundant code and `!important`.

File: style.css
Comment (bolah2009) [inline style.css]: #### `REQUIRED`

### Consider deleting this.

```suggestion
```

Using `justify-content: center;` for `display: block;` is redundant.
`justify-content` is a `Flexbox` (`display: flex;`) and `Grid Layout` (`display: grid;`) property only. 

Disabling it as shown below have no effect. (including `margin-top: 0 !important;` and `padding-top: 0 !important;`)

apple-disable

### Learn more about [`justify-content` here](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content)

File: style.css
Comment (bolah2009) [inline style.css]: #### `REQUIRED`

### Consider deleting this.

```suggestion
```

See comment on using `justify-content` for `display` property other than `Flexbox` (`display: flex;`) and `Grid Layout` (`display: grid;`). :blush:

File: style.css
Comment (bolah2009) [inline style.css]: #### `REQUIRED`

### Consider deleting this.

```suggestion
```

Like, `justify-content`, using `align-items: center;` for `display: inline-block;` is redundant.
`align-items` is a Flexbox (`display: flex;`) and Grid Layout (`display: grid;`) property only. 

Try disabling it with the `Developer Tools` on the browser.

### Learn more about [`align-items` here](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items)

File: style.css
Comment (bolah2009) [inline style.css]: #### `REQUIRED`

```suggestion
```

See comment on using `justify-content` for `display` property other than `Flexbox` (`display: flex;`) and `Grid Layout` (`display: grid;`). :blush:

File: style.css
Comment (bolah2009) [inline style.css]: #### `REQUIRED`

```suggestion
```

See comment on using `align-items` for `display` property other than `Flexbox` (`display: flex;`) and `Grid Layout` (`display: grid;`). :blush:

File: style.css
Comment (bolah2009) [inline style.css]: #### `REQUIRED`

```suggestion
```

Like, `justify-content`, using `align-content: center;` for `display: block;` is redundant.
`align-content` is a Flexbox (`display: flex;`) and Grid Layout (`display: grid;`) property only. 

Try disabling it with the `Developer Tools` on the browser.

### Learn more about [`align-content` here](https://developer.mozilla.org/en-US/docs/Web/CSS/align-content)

File: style.css
Comment (bolah2009) [inline style.css]: #### `REQUIRED`

### :bug: This is what we need to change to fix the unwanted gutter at the top of the page. :smile: 

gutter-apple-unwanted

```suggestion
    position: absolute;
    left: <any length value>;
```

#### See demo below:

fix-apple-bug

### Learn more about [**positioned elements** here](https://developer.mozilla.org/en-US/docs/Web/CSS/position)

**Review by bolah2009 — CHANGES_REQUESTED** [review]
### Awesome @kilonzi 

#### You did a nice job with the [clone](https://raw.githack.com/kilonzi/Microverse-Apple/df66b5f8cd62c06672a3513e891860d1c0e804b5/index.html) :ok_hand: Uses CSS backgrounds and gradients effectively.

However, there is a :bug: to fix before a :+1: :smile: See suggestions in the comment below ([especially **this**](#discussion_r319752463)). 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Review by bolah2009 — APPROVED** [review]
# Nice Job @kilonzi  

### Good Job implementing the suggested changes. It's a :+1: from me...

#### The [clone](https://raw.githack.com/kilonzi/Microverse-Apple/f496c9fd5877fd6bf3feaf7c102abe605638be4a/index.html) looks even better now... :smile: :ok_hand: 

Consider making the suggested changes in these comments ( [one](https://github.com/kilonzi/Microverse-Apple/pull/3/files#r319751542) and [two](https://github.com/kilonzi/Microverse-Apple/pull/3/files#r319751345)). They are redundant (useless) CSS property in this context that has nothing to do with `display: inline-block;` as explained in the comments. **Please delete.**

## Please do this before merging... :pray: 

Good luck with your next project. :star2: 

Cheers
[Bola](https://bolabuari.com/)  

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Pull request merged**
